src/chainstate/stacks/db/mod.rs

```diff
@@ -1327,35 +1320,31 @@ impl StacksChainState {
     /// parameters, but with a block limit
     pub fn reopen_limited(
```

Comment:
There's no longer a need to call this `reopen_limited()` if an execution cost is not going to be given to constrain the execution budget.

Reply:
Yep, that's correct-- we still want to `reopen`.